### ISSUE TITLE
Accuracy on the comment classification does not reach the reported value?

### ISSUE THREAD

**Issue by anoynomous101010** [ICSE-DOME/DOME#1]
Thanks for your great work. 
I have tried to train the comment classifier based on your instrutions., i.e., finetuned the dataset using CodeBert. However,
the test results on the test set are:
![image](https://user-images.githubusercontent.com/126099163/220698467-62ae28a2-12d9-4f56-82d5-2f5f30373d10.png)
which does not reach the accuracy repoted in the paper.
Could you please give some advice or release the pkl file of the comment classifier?
Looking forward to your reply!

**Comment by ICSE-DOME** [comment]
Hi, 
for the comment classifier, we manually annotate 20k code-comment data into six intent categories. Since some comments are very vague, and it is difficult to determine their intent, we remove these comments (i.e. noisy data) and split the rest of the data into 10 folds (9 folds for training and 1 fold for testing).

I have uploaded the 10-fold dataset and the well-trained checkpoint file on Google Drive,  you can download them from this link [https://drive.google.com/file/d/15WUpuxyzy21nTuECvL4BFxYbCOihue6b/view](https://drive.google.com/file/d/15WUpuxyzy21nTuECvL4BFxYbCOihue6b/view). 
The results on the test set should be:
![image](https://user-images.githubusercontent.com/112490115/221194583-daaf8d72-61f5-432e-95ae-587da7ce9ba5.png)

**Comment by anoynomous101010** [comment]
Hi, thanks for the released pkl file and the dataset. I still have a question about the dataset.
Is the 10-fold dataset finally pre-processed, i.e., clean version? And what is the final amount of training and test data in your dataset? I found that in the "label" attribute of the train_0.json, there are some values "convert", not belonging to the six categories. Could you please check and help me clarify?
Thanks in advance!

**Comment by hungkien05** [comment]
@anoynomous101010 Hi, have you found any solutions for this problem ? Besides "convert", I also notice there are labels named "what-why" and "what-how" .
